src/azure-cli/azure/cli/command_modules/backup/custom.py

```diff
@@ -179,11 +178,10 @@ def set_policy(client, resource_group_name, vault_name, policy, policy_name):
     return client.create_or_update(vault_name, resource_group_name, policy_name, policy_object)
 
 
-def create_policy(client, resource_group_name, vault_name, name, policy):
+def create_policy(client, resource_group_name, vault_name, name, policy, backup_management_type="AzureIaasVM"):
```

Comment:
don't pass this parameter. hardcode instead. policy_object.properties.backup_management_type = "AzureIaasVM"

Reply:
okay